### ISSUE TITLE
broken repo path for Amazon Linux

### ISSUE THREAD

**Issue by jworl** [saltstack/salt-bootstrap#1547]
### Description of Issue/Question
receiving 404 not found
https://repo.saltproject.io/yum/amazon/latest/x86_64/latest/repodata/repomd.xml: [Errno 14] HTTPS Error 404 - Not Found

This path does not exist anymore. Manually looking, it seems "https://repo.saltproject.io/yum/amazon/" directory only contains the "2" folder. "latest" does not exist.

### Setup
```
NAME="Amazon Linux AMI"
VERSION="2018.03"
ID="amzn"
ID_LIKE="rhel fedora"
VERSION_ID="2018.03"
PRETTY_NAME="Amazon Linux AMI 2018.03"
ANSI_COLOR="0;33"
CPE_NAME="cpe:/o:amazon:linux:2018.03:ga"
HOME_URL="http://aws.amazon.com/amazon-linux-ami/"
Amazon Linux AMI release 2018.03
```

### Steps to Reproduce Issue
```
sudo curl -fsSL https://bootstrap.saltproject.io -o install_salt.sh
sudo sh install_salt.sh git -P -x python3 -j "`cat /tmp/minion_conf`" -D
```
(debug logs attached)
[bootstrap-salt.log](https://github.com/saltstack/salt-bootstrap/files/6212833/bootstrap-salt.log)

### Versions and Systems
```
NAME="Amazon Linux AMI"
VERSION="2018.03"
ID="amzn"
ID_LIKE="rhel fedora"
VERSION_ID="2018.03"
PRETTY_NAME="Amazon Linux AMI 2018.03"
ANSI_COLOR="0;33"
CPE_NAME="cpe:/o:amazon:linux:2018.03:ga"
HOME_URL="http://aws.amazon.com/amazon-linux-ami/"
Amazon Linux AMI release 2018.03
```

**Comment by bryceml** [comment]
amazonlinux 1 is end of life, and the salt packages for it have been moved to https://archive.repo.saltproject.io

You can use `-R archive.repo.saltproject.io` as an argument to the bootstrap script if you want to use them.  They are unsupported and all contain CVEs.

**Comment by jworl** [comment]
Ah. I understand. Thanks for the quick response! I just tried this with `-R archive.repo.saltproject.io` , but I receive the same error. Are there additional flags which I must set for this to work properly? I glanced through the documentation, but nothing stood out to me.

`sudo sh install_salt.sh -R "archive.repo.saltproject.io" -j "$(cat /tmp/minion_conf)" -D`

[bootstrap-salt-R-flag.log](https://github.com/saltstack/salt-bootstrap/files/6213001/bootstrap-salt-R-flag.log)
